### DISCUSSION TITLE
自动翻译源码中的文档

### DISCUSSION THREAD

**Discussion by fmnisme** [YiiGuxing/TranslationPlugin#4324] [Ideas]
<img width="838" alt="image" src="https://github.com/YiiGuxing/TranslationPlugin/assets/5197501/1abc5a6d-00d0-4c9e-a390-a1bc99cbad55">
需要点击z这里才会返回文档，可以变成自动翻译吗

**Comment by YiiGuxing** [comment]
已经有相关的需求了： #1760
